Automattic/woocommerce-payments
# Remove reduntant text after a progressive account is onboarded

Fixes #8278 

#### Changes proposed in this Pull Request

Remove reduntant message after a progressive account is onboarded.
<img width="412" alt="Screenshot 2024-03-18 at 14 49 35" src="https://github.com/Automattic/woocommerce-payments/assets/82826872/7093c508-d453-42ab-ade1-8d9fc85e82e7">









#### Testing instructions
1. For your current account, go to server db, and change the `account_type` field in `wcpay_accounts`  table, from `wcpay` to `wcpay_other`
2. go to WCPay Dev and clear account cache
3. Go to `Payments` and press Finish button 
4. Onboard on Stripe.
5. press the return to dev shared account
6. check if you can see the success modal
<img width="730" alt="Screenshot 2024-03-18 at 14 54 43" src="https://github.com/Automattic/woocommerce-payments/assets/82826872/d728ddef-f8b7-40cc-932c-06287df754d0">
7. The message "You're ready to start selling" shouldn't be shown amymore


https://github.com/Automattic/woocommerce-payments/assets/82826872/a49e2f26-d9ac-439e-a3a8-801056d1e1c5







*

-------------------


**Post merge**
